test/Microsoft.ML.Core.Tests/UnitTests/TestEntryPoints.cs

```diff
@@ -2521,5 +2521,56 @@ public void EntryPointPrepareLabelConvertPredictedLabel()
                 }
             }
         }
+
+        [Fact]
+        public void EntryPointTreeLeafFeaturizer()
+        {
+            var dataPath = GetDataPath(@"adult.tiny.with-schema.txt");
+            var inputFile = new SimpleFileHandle(Env, dataPath, false, false);
+            var dataView = ImportTextData.ImportText(Env, new ImportTextData.Input { InputFile = inputFile }).Data;
+            var cat = Categorical.CatTransformDict(Env, new CategoricalTransform.Arguments()
+            {
+                Data = dataView,
+                Column = new[] { new CategoricalTransform.Column { Name = "Categories", Source = "Categories" } }
+            });
+            var concat = SchemaManipulation.ConcatColumns(Env, new ConcatTransform.Arguments()
+            {
+                Data = cat.OutputData,
+                Column = new[] { new ConcatTransform.Column { Name = "Features", Source = new[] { "Categories", "NumericFeatures" } } }
+            });
+
+            var fastTree = FastTree.FastTree.TrainBinary(Env, new FastTreeBinaryClassificationTrainer.Arguments
+            {
+                FeatureColumn = "Features",
+                NumLeaves = 80,
+                LabelColumn = DefaultColumnNames.Label,
+                TrainingData = concat.OutputData
+            });
+
+            var combine = ModelOperations.CombineModels(Env, new ModelOperations.PredictorModelInput()
+            {
+                PredictorModel = fastTree.PredictorModel,
+                TransformModels = new[] { cat.Model, concat.Model }
+            });
+
+            var treeLeaf = TreeFeaturize.Featurizer(Env, new TreeEnsembleFeaturizerTransform.ArgumentsForEntryPoint
+            {
+                Data = dataView,
+                PredictorModel = combine.PredictorModel
+            });
+
+            var view = treeLeaf.OutputData;
+            Assert.True(view.Schema.TryGetColumnIndex("Trees", out int col));
+            VBuffer<float> features = default(VBuffer<float>);
+            using (var curs = view.GetRowCursor(c => c == col))
+            {
+                var getter = curs.GetGetter<VBuffer<float>>(col);
+                while (curs.MoveNext())
+                {
+                    getter(ref features);
+                    Assert.True(features.Count > 0);
```

Comment:
>features.Count [](start = 32, length = 14)

Can we make a few more definite statements about this vector to make the test a bit stronger? For example: It should not only have some non-zero entries, we'd expect it to be a particular length, etc., right?